large_language_model/nemo/config.sh

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can we have an example config.sh file with all these values filled out so users know what the paths should point to or what the contents for these values should look like?